src/rpc/rawtransaction.cpp

```diff
@@ -410,9 +410,9 @@ UniValue verifytxoutproof(const UniValue& params, bool fHelp)
 
 UniValue createrawtransaction(const UniValue& params, bool fHelp)
 {
-    if (fHelp || params.size() < 2 || params.size() > 3)
+    if (fHelp || params.size() < 2 || params.size() > 4)
         throw runtime_error(
-            "createrawtransaction [{\"txid\":\"id\",\"vout\":n},...] {\"address\":amount,...} ( locktime )\n"
+            "createrawtransaction [{\"txid\":\"id\",\"vout\":n},...] {\"address\":amount,...} ( locktime ) ( expiryheight )\n"
```

Comment:
In Bitcoin Core, this positional argument is the boolean `replaceable`, allowing BIP 125 to be configured (which enables a transaction to be later replaced by one with higher fees). Given that we are highly unlikely to ever bring in BIP 125 (both because it creates a distinguisher wrt fees, and we have transaction expiry which we are configuring with the same positional), I'm content with our addition here.